doc/news/_preparation_next_release.md

```diff
@@ -128,6 +128,12 @@ you up to date with the multi-language support provided by Elektra.
 
 ## Build
 
+## Other
```

Comment:
I think it would make sense to change the header name to `### Augeas` and move the text below the `## Plugins` section.